lib/archethic_web/live/transaction_details_live.ex

```diff
@@ -72,7 +70,7 @@ defmodule ArchEthicWeb.TransactionDetailsLive do
 
   defp handle_transaction(
          socket,
-         tx = %Transaction{address: address, data: %TransactionData{content: content}}
+         tx = %Transaction{address: address, data: %TransactionData{content: _content}}
```

Comment:
I think you can also clean this one

Reply:
transaction.data.content we are using it in leex file
![Screenshot 2021-09-20 at 2 37 02 PM](https://user-images.githubusercontent.com/23115337/133979221-e5c65bdf-dda0-4f2b-bcee-3d8ee30995c4.png)

Reply:
We are already passing the transaction to the view, but here the pattern matching was used to determine if you show the content depending on its size, for performance issue.
So if you want to use this var, you should set this variable into the alpine.js data also.

Reply:
ok, i have removed "data: %TransactionData{content: _content"